sbt-plugin-test/multiTest/js/src/test/resources/consoleWriter.js

```diff
@@ -0,0 +1 @@
+console.log("Breaking message")
```

Comment:
And maybe also here by the way. I just want to make sure that if someone stumbles over the file it does take minimal effort to find its purpose.